### ISSUE TITLE
which dataset uses the pertained model from kinetics?

### ISSUE THREAD

**Issue by beckybai** [mit-han-lab/temporal-shift-module#33]
Hi, thanks for the code release. In your first version of Arxiv paper, 

> We then fine-tuned the model to other target datasets like Something- Something [12], UCF101 [34], and HMDB51 [22]

In the most current version
> For most of the datasets, the model is fine-tuned from ImageNet pre-trained weights; while HMDB-51 [26] and UCF-101 [40] are too small and prone to over-fitting [48], we followed the common practice [48, 49] to fine-tune from Kinetics [25] pre-trained weights and freeze the Batch Normalization [22] layers. 

Which dataset is trained using the pre-trained model to get the score reported in the paper? Jester, UCF101 and HDMB? Are the parameters set for Jester and HDMB the same as UCF101?

Thanks again.

**Comment by beckybai** [comment]
One more thing, when you add the optic flow input, I am wondering if the parameters should be double? Table 3 in the paper shows that both models have the same parameters. Thanks. 😃

**Comment by tonylins** [comment]
We used Kinetics pre-trained model on HMDB-51 and UCF-101, while ImageNet pre-trained model on other datasets. We used an identical model (except for the number of input channels) for optical flow.

**Comment by beckybai** [comment]
I'm appreciate for the quick reply. What does the 'identical' mean? Is it structural identical or weight identical? Do you train RGB first and then train flow separately and treat it as an ensemble model? Or you combine those two as the input to the NN? Thanks.

**Comment by tonylins** [comment]
Hi, I trained RGB and Flow models separately, following the common implementation. During test, I just calculate the average of the predictions of RGB and Flow.

**Comment by ucasiggcas** [comment]
hi,dear
could you pls supply the txt file or where I can find ?
thx
**kinetics/labels/val_videofolder.txt**
https://github.com/mit-han-lab/temporal-shift-module/issues/42
